docker-compose.yml

```diff
@@ -811,6 +811,57 @@ services:
         max-size: 20m
         max-file: '5'
 
+  http-agent-cert-sidecar:
+    image: dojot/cert-sidecar:${DOJOT_VERSION}
+    depends_on:
+      - x509-identity-mgmt
+    environment:
+      CERT_SC_APP_SIDECAR_TO: 'http-agent'
+      CERT_SC_LOG_CONSOLE_LEVEL: 'info'
+      CERT_SC_CERTS_HOSTNAMES: '["http-agent", "localhost"]'
+      HOSTNAME: http-agent
+      CERT_SC_CERTS_FILES_BASEPATH: /certs
+      CERT_SC_CERTS_FILES_CA: ca.crt
+      CERT_SC_CERTS_FILES_CERT: http-agent.crt
+      CERT_SC_CERTS_FILES_KEY: http-agent.key
+      CERT_SC_CERTS_FILES_CRL: ca.crl
+      CERT_SC_CRON_CRL_TIME: '0 */1 * * * *'
+    volumes:
+      - http-agent-cert-side-volume:/certs
+    restart: always
+    logging:
+      driver: json-file
+      options:
+        max-size: 20m
+        max-file: '5'
+
+  http-agent:
+    image: dojot/http-agent:${DOJOT_VERSION}
+    depends_on:
+      - kafka
+      - data-broker
+      - auth
+      - http-agent-cert-sidecar
+      - device-manager
+      - certificate-acl
+    ports:
+      - 8080:3000
+    restart: always
+    environment:
+      HTTP_AGENT_HTTPS_CERT: '/certs/http-agent.crt'
+      HTTP_AGENT_HTTPS_KEY: '/certs/http-agent.key'
+      HTTP_AGENT_HTTPS_CA: '/certs/ca.crt'
+      HTTP_AGENT_SECURITY_CRL: '/certs/ca.crl'
+      HTTP_AGENT_SECURITY_CERT_DIRECTORY: '/certs'
+      HTTP_AGENT_LOG_CONSOLE_LEVEL: 'info'
+    volumes:
+      - http-agent-cert-side-volume:/certs
```

Comment:
Is this right? The same volume was mapped for `http-agent-cert-sidecar`.
If is it right you should apply here the same aesthetic comment that i mentioned before.